config/locales/de.date_ranges.yml

```diff
@@ -2,7 +2,14 @@
 
 de:
   date_range:
-    to: "bis"
+    errors:
+      starts_in_past: "Anfangs-Datum darf nicht in der Vergangenheit liegen"
+      end_in_past: "darf nicht in der Vergangenheit liegen"
+      end_before_start: "kann nicht vor Start-Datum liegen"
+      no_timespan: "Bitte mindestens eine Zeitspanne auswählen"
```

Comment:
Do translations need to be double-quoted? Just because everyone obviously does it this way does not necessarily mean they needs to be.